### DISCUSSION TITLE
Docker SSL

### DISCUSSION THREAD

**Discussion by vdiaza** [chartbrew/chartbrew#109] [Q&A]
Hi! 
I have been playing with this app and I loved it!
I was wondering if anyone can help guiding me on setup the [docker](https://hub.docker.com/r/razvanilin/chartbrew) with SSL enabled?

I was able to deploy it on a digitalocean droplet but I am having trouble on the steps to enable SSL since I am not very familiar with docker. I looked through the web but I could not find anything with a docker already built.

Thank you!

**Comment by razvanilin** [comment]
Hey @vdiaza many apologies for the delayed reply, my notifications were apparently disabled for the discussions.

Have you managed to make any progress with getting the SSL working? I think the issue is not on the Docker side. You will need to set up the hosts with your webserver to enable SSL. 

Can you let me know what webserver you are using? (e.g. Apache, nginx)
